lua/cmake-tools/init.lua

```diff
@@ -265,7 +307,7 @@ function cmake.clean(callback)
     return log.error(result.message)
   end
 
-  local args = { "--build", config.build_directory.filename, "--target", "clean" }
+  local args = { "--build", '"' .. config.build_directory.filename .. '"', "--target", "clean" }
```

Comment:
The "" is for Windows?

Reply:
Yes

Reply:
This isn't really necessary, It messes up the file paths. We can just ask users to simply not use paths with space which get translated to: `'C:\Program Files\....'` Otherwise we need to do spearate file path handling operations for windows, and `:CMakeArgs' (https://github.com/Civitasv/cmake-tools.nvim/issues/90#issuecomment-1703959109) would not work on Windows as of now.